redisplus/helpers.py

```diff
@@ -34,3 +34,14 @@ def parseToList(response):
         else:
             res.append(None)
     return res
+
+
+def decodeDicKeys(obj):
+    """Decode the keys of the given dictionary with utf-8."""
+    obj_new = {}
```

Comment:
import copy
o = copy.copy(obj)
for k in obj.keys():
    if isinstance(k, bytes):
        o[k.decode('utf-8')] = o[k]
        o.pop()
return o

Reply:
What's the meaning of `o.pop()` here?

Reply:
o.pop pulls the key out of the dictionary. Since we're fixing the binary key and reseeding it - we're deleting the old key, right after we set the new one.